enzy_htp/_interface/rosetta_interface.py

```diff
@@ -1498,5 +1500,173 @@ def build_cartesian_ddg_engine(
+                muts.append( mut_name )
+            if not muts:
+                return stru
+            content.extend("""</MOVERS>
+<PROTOCOLS>""".split())
+            
+            for mm in muts:
+                content.append(f"\t<Add mover_name=\"{mm}\"/>")
+
+            content.extend("""</PROTOCOLS>
+<OUTPUT />
+</ROSETTASCRIPTS>""".splitlines())
+            
+            script_file = f"{work_dir}/fix_remodel_script.xml"
+            pdb_infile = f"{work_dir}/fix_remodel.pdb"
+            pdb_outfile = f"{work_dir}/fix_remodel_0001.pdb"
+
+            fs.write_lines(script_file, content)
+            sp = PDBParser()
+            sp.save_structure(pdb_infile, stru)
+
+            opts = [
+                '-in:file:s', pdb_infile,
+                '-parser:protocol', script_file,
+                '-out:path:all', work_dir
+            ]
+            self.run_rosetta_scripts( opts )
+
+            if not Path(pdb_outfile).exists():
+                err_msg:str=f"The output file '{pdb_outfile}' was not created. Error during loop reconstruction fixing protocol."
+                _LOGGER.error( err_msg )
+                raise ValueError( err_msg )
+
+
+            fs.safe_rm(script_file)
+            fs.safe_rm(pdb_infile)
+            fs.safe_rm(pdb_outfile)
+
+            return sp.get_structure( pdb_outfile )
+
+
+        fs.safe_mkdir( work_dir )
+
+        translate_structure( stru, end_naming='rosetta' )
+        
+        all_chains:List[Chain] = stru.non_aa_chains
+
+        for chain in stru.aa_chains:
+            single_chain_stru = Structure([chain])
+            seq_res = single_chain_stru.seqres_sequence
+            mr : SeqRes
+            for mr in missing_residues:
+                if mr.chain == chain.name:
+                    seq_res.append(mr)
+            seq_res.sort()
+
+
+            existing = 1
+            content = list()
+            for sidx,sr in enumerate(seq_res):
+                if sr.missing:
+                    line = f"0 X L PIKAA {sr.one_letter()}"
+                else:
+                    borders_loop:bool = (
+                        sidx > 0 and seq_res[sidx-1].missing 
+                    ) or (
+                        sidx < len(seq_res) - 1 and seq_res[sidx+1].missing                
+                    )
+                    if borders_loop:
+                        line = f"{existing} {sr.one_letter()} L PIKAA {sr.one_letter()}"
+                    else:
+                        line = f"{existing} {sr.one_letter()} ."
+                    existing += 1
+            
+                content.append(line)
+
+
+            remodel_file:str = f"{work_dir}/input.remodel"
+            fs.write_lines( remodel_file, content ) 
+            script_file:str=f"{work_dir}/remodel_script.xml"
+            pdb_file = f"{work_dir}/remodel_input.pdb"
+            pdb_outfile = f"{work_dir}/remodel_input_0001.pdb"
+            write_xml_input( script_file )
```

Comment:
Is this defined anywhere?

Reply:
Nope. left it in by accident. It is now removed.